src/components/ListItem.jsx

```diff
@@ -18,17 +18,24 @@ export function ListItem({ item, listToken }) {
 		}
 	};
 
+	const isRecentlyPurchased =
+		sub(new Date(), { days: 1 }) < item.dateLastPurchased;
+
 	return (
-		<li className="ListItem">
+		<li
+			className="ListItem"
+			title={isRecentlyPurchased && 'You recently purchased this item'}
```

Comment:
I think you should only add the `title` attribute once, and since the `input` is the child of this `li` (and it also already has a `label`), let's just add it the `li` and remove it for the `input`. 😊

Reply:
I thought I hovered on the label and a description didn't show when I had it on the label so I just added the title on the two. I've updated it now. 😅